spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -61,7 +70,7 @@ deployment to fail
 ```
 cf push dataflow-server --no-start -p spring-cloud-dataflow-server-cloudfoundry-{project-version}.jar
 cf bind-service dataflow-server redis
-cf bind-service dataflow-server rabbit
```

Comment:
nope, seems we don't need it for that. So I think the update in this PR is legit